### ISSUE TITLE
Deprecation message with PHP 8.1

### ISSUE THREAD

**Issue by jokaorgua** [Nyholm/psr7#195]
Hello .
I have some deprecation messages when using lib 1.5 and PHP 8.1
```
Method "Psr\Http\Message\ServerRequestInterface::withUploadedFiles()" might add "static" as a native return type declaration in the future. Do the same in implementation "Nyholm\Psr7\ServerRequest" now to avoid errors or add an explicit @return annotation to suppress this message.

  1x: Method "Psr\Http\Message\ServerRequestInterface::withCookieParams()" might add "static" as a native return type declaration in the future. Do the same in implementation "Nyholm\Psr7\ServerRequest" now to avoid errors or add an explicit @return annotation to suppress this message.

  1x: Method "Psr\Http\Message\ServerRequestInterface::withQueryParams()" might add "static" as a native return type declaration in the future. Do the same in implementation "Nyholm\Psr7\ServerRequest" now to avoid errors or add an explicit @return annotation to suppress this message.

  1x: Method "Psr\Http\Message\ServerRequestInterface::getParsedBody()" might add "array|object|null" as a native return type declaration in the future. Do the same in implementation "Nyholm\Psr7\ServerRequest" now to avoid errors or add an explicit @return annotation to suppress this message.

  1x: Method "Psr\Http\Message\ServerRequestInterface::withParsedBody()" might add "static" as a native return type declaration in the future. Do the same in implementation "Nyholm\Psr7\ServerRequest" now to avoid errors or add an explicit @return annotation to suppress this message.
```

**Comment by jokaorgua** [comment]
@Nyholm could you take a look at pull request ?

**Comment by Nyholm** [comment]
This is fixed in 1.5.1
